### PR TITLE
Include reform

### PR DESCRIPTION
**Pull request by adrienpacifico**
Modifications in france-data, plugin-agregates and survey-manager

### CODE REVIEW

File: openfisca_france_data/surveys.py [hunk 1]
```diff
@@ -33,21 +33,33 @@
...
+    def init_from_data_frame(self, input_data_frame = None,reform = None,
+            tax_benefit_system = None, used_as_input_variables = None,
             year = None):
-
         if tax_benefit_system is None:
-            TaxBenefitSystem = openfisca_france_data.init_country()
-            tax_benefit_system = TaxBenefitSystem()
+            log.info("="*10, reform , "="*10)
+            if reform is None:
+                FranceDataTaxBenefitSystem = openfisca_france_data.init_country()
```
Comment (benjello) [inline openfisca_france_data/surveys.py]: Do not introduce new names

File: openfisca_france_data/surveys.py [hunk 2]
```diff
@@ -33,21 +33,33 @@
...
+                france_data_tax_benefit_system = FranceDataTaxBenefitSystem()
+                tax_benefit_system = france_data_tax_benefit_system
+            else:
+
+                FranceDataTaxBenefitSystem = openfisca_france_data.init_country()
```
Comment (benjello) [inline openfisca_france_data/surveys.py]: Do not introduce new names

**Comment by adrienpacifico** [comment]
@cbenz  told me to change the name because of possible confusions between france_data and normal tax benefit system (which happened to me).

**Comment by benjello** [comment]
Use reference vs reform or something more general

**Comment by adrienpacifico** [comment]
The problem is not reform vs. reference but TaxBenfitSystem vs. DataTaxBenefitSystem.

**Comment by benjello** [comment]
There is no "data" in TaxBenefitSystem. There are SurveyScenario and classical test-case Scenario.
Clarify your problem, your goal and your implementation. I will look into it ASAP.
And write a test of what we be the result you want.

**Comment by adrienpacifico** [comment]
openfsica_france_data/model/base.py create its own version of openfisca france tax benefit system and it is a Reform object. Thus I call that the  DataTaxBenefitSystem.

My problem : I want an easy way to run a reform instead of the reference tax benefit system.
My goal : Solve my problem ?  It's done.

My implementation : Changed test_agregates.py such that I can run a reform if I want to. You seams to want a dedicated file to do that, so I changed for test_agregates_reform.py on my computer for next commit but I am not convinced since it does the same thing as test_agregates but it include an option to include a reform.


 I made some modifications to include any reform when with test_agregates, everything seams to work.


For the FranceDataTaxBenefitSystem vs. TaxBenefitSystem issue, I really don't care it was Chistophes' suggestion, even though it makes naming a bit more explicit and might prevent future users to confound original france tax benefit system  to france data tax benefit system if the second is modified (which is for now not the case, but the implementation seams that it might be planned).

No need for me to look at it asap, just need to do not diverge too much before I leave cause it can be difficult for me to sort things correctly from Marseille.

**Comment by benjello** [comment]
Rebase and resubmit

**Comment by benjello** [comment]
Please close #99 
Please rebase and resubmit this reform.
I can do it by hand but you will end up with a mess

**Pull request closed without merging**